### DISCUSSION TITLE
Do we have api to track the cluster status?

### DISCUSSION THREAD

**Discussion by godlockin** [milvus-io/milvus#6740] [Q&A and General discussion] [answered]
Do we have api to track the cluster status? for instance, all nodes are ready, the size of data storage, cpu/memory status .etc for monitor & devops requirements?

**Comment by xiaofan-luan** [comment, marked as answer]
> Do we have api to track the cluster status? for instance, all nodes are ready, the size of data storage, cpu/memory status .etc for monitor & devops requirements?

yep, that's a good one. We already supported prometheus and grafana, the document is on the way.
We will also offer Apis for cluster status in Milvus 2.0 release

**Reply by godlockin** [reply]
yeah, that's quite nice to support prometheus & grafana, but the direct apis would be more helpful on cluster (node) status check on both auto-devops jobs and manually check then.

waiting for the new apis then 😁
